doc/source/basic-use.rst

```diff
@@ -1,153 +1,259 @@
+for iocage usage.
+
+Once iocage is ready with an active zpool, users are able to immediately
+begin downloading FreeBSD releases for jail creation.
+
+.. index:: Fetch a release
+.. _Fetch a Release:
+
+Fetch a Release
+---------------
 
-Simply run:
+iocage now needs to fetch a RELEASE, which is used to create jails. By
+default, typing :command:`iocage fetch` opens a menu for the user to
+choose which release to download, as seen in this example:
 
-``iocage fetch``
+.. code-block:: none
 
-If a specific RELEASE is required run:
+ # iocage fetch
+ [0] 9.3-RELEASE (EOL)
+ [1] 10.1-RELEASE (EOL)
+ [2] 10.2-RELEASE (EOL)
+ [3] 10.3-RELEASE
+ [4] 11.0-RELEASE
 
-``iocage fetch release=9.3-RELEASE``
+ Type the number of the desired RELEASE
+ Press [Enter] to fetch the default selection: (default)
+ Type EXIT to quit: 4
 
-In case a specific download mirror is required simply run:
+Once the desired RELEASE is downloaded, the most recent patches are also
+applied to it.
 
-``iocage fetch ftphost=ftp.hostname.org``
+:command:`iocage fetch` also has a number of options and properties for
+users to fine-tune the functionality of the command.
 
-You can also specify a ftp directory to fetch the base files from:
+If a specific RELEASE is required, type:
 
-``iocage fetch ftpdir=/dir/``
+:samp:`# iocage fetch release=10.3-RELEASE`
 
-Create a jail
+If a specific download mirror is required, type:
+
+:samp:`# iocage fetch ftphost=ftp.hostname.org`
+
+:command:`fetch` can also pull from a specific ftp directory:
+
+:samp:`# iocage fetch ftpdir=/dir/`
+
+.. index:: Basic Jail Creation
+.. _Create a Jail:
+
+Create a Jail
 -------------
 
-There are three supported basic jail types: full, clone and base jail. In addition to these three 
-there are two more which are discussed later (empty and templates).
-Depending on requirements the `create` subcommand can be tweaked to create any of the three types.
-By default iocage will create a fully independent jail of the current host's RELEASE and set the TAG property to todays date.
+With a release downloaded, iocage is now able to create jails. There are
+five types of jails, three basic and two advanced. Basic jail types are
+**full**, **clone**, and **base** jails. Advanced jails types are
+**empty** and **templates**, but these are discussed in the
+:ref:`Jail Types` section, along with more detailed descriptions of the
+basic jail types.
+
+Depending on the user's requirements, the :command:`create` subcommand
+can be tweaked to create any of the three basic types. By default,
+iocage creates a fully independent jail of the current host's RELEASE,
+setting the TAG property to the current date.
+
+The :command:`create` command can be used to quickly create a basic
+jail:
+
+:samp:`# iocage create -r 11.0-RELEASE`
 
-Creating a jail is real simple, just run:
+This basic jail is fully independent.
 
-``iocage create``
+To create a full jails with multiple clones, type:
```

Comment:
Should be stated as to create multiple jails. They're all clones, but using `--count` does not cause a relationship link between them. They are all individual clones from the RELEASE.